client/packages/invoices/src/OutboundShipment/api/hooks/document/useOutboundUpdateInvoiceTax.ts

```diff
@@ -0,0 +1,27 @@
+import { useOutboundApi } from './../utils/useOutboundApi';
+import { useMutation, useQueryClient } from '@openmsupply-client/common';
+import { useCallback } from 'react';
+import { useOutbound } from './useOutbound';
+
+export const useOutboundUpdateInvoiceTax = () => {
```

Comment:
Created to match how hook is being used in Outbounds and can be deleted once invoice line tax has been decided...